WordPress/pattern-directory
# Creator: Add welcome guide on first load.

This PR closes #96, adding a welcome guide for new users. 

I have added the component into the current _Creator_ which will most likely be changed when #59 is resolved. I think it's fine to implement it here and then move it over later.

**To Do:**
This needs integration with the final `pattern-creator`.



### Screenshots

| 1 | 2 | 3 | 4 | Screencast | 
| --- | --- | --- | --- | --- |
| ![](https://d.pr/i/OgjPIy.png) | ![](https://d.pr/i/1vEPKz.png) | ![](https://d.pr/i/k9SmZO.png) | ![](https://d.pr/i/zQVhZF.png) | ![](https://d.pr/i/Ajf8e7.gif) |

### How to test the changes in this Pull Request:

1. Create a page with the title `new-pattern`
2. Navigate to `/new-pattern`
3. Expect to see the new welcome guide (same as screenshot)
4. Continue through the welcome guide until complete
5. Reload the page
6. Expect to not see the welcome guide
